engine/src/test/java/io/camunda/zeebe/engine/processing/resource/ResourceDeletionTest.java

```diff
@@ -171,13 +172,14 @@ public void shouldEvaluateLatestVersionIfPreviousVersionIsDeleted() {
             tuple(BpmnElementType.PROCESS, ProcessInstanceIntent.ELEMENT_COMPLETED));
 
     assertThat(
-        RecordingExporter.decisionEvaluationRecords(DecisionEvaluationIntent.EVALUATED)
-            .withProcessInstanceKey(processInstanceKey)
-            .withDecisionId("jedi_or_sith")
-            .getFirst())
+            RecordingExporter.decisionEvaluationRecords(DecisionEvaluationIntent.EVALUATED)
+                .withProcessInstanceKey(processInstanceKey)
+                .withDecisionId("jedi_or_sith")
+                .getFirst())
```

Comment:
❓ Is your Save-Action plugin not functioning correctly? It should produce the same results as spotless.

Reply:
Save-actions worked fine. I switched to InelliJ Ultimate Edition and it messed up some settings, one of which is the google-java-format plugin. I forgot [this bug](https://github.com/google/google-java-format/issues/787) existed. But it's all good now.